### PR TITLE
Improve example app

### PR DESCRIPTION
**Pull request by steveklebanoff**
Adds some improvements to the example app:
- Show only appropriate links based on logged in status
- Fix logout link
- Add bar at top telling logged in status

When I first logged into the example app, I was confused that I was even logged in because the page I was redirected to looked the same as when I was logged out.  Also, logging out seemed broken since the page didn't refresh automatically.

### CODE REVIEW

File: example_app/web/templates/layout/app.html.eex [hunk 1]
```diff
@@ -25,7 +27,5 @@
       </main>
 
     </div> <!-- /container -->
-    <script src="<%= static_path(@conn, "/js/jquery.min.js") %>"></script>
```
Comment (steveklebanoff) [inline example_app/web/templates/layout/app.html.eex:28]: I know it's a best practice to have the JS load at the bottom of the page, but it was easiest to move it to the `<head>` to be able to put in some inline JS to bind the button in the template.  

Alternatively I could have put the button binding in app.js or a template-specific JS file. I'm down for whatever on this, not sure how much we care about JS cleanliness in this example app

File: example_app/web/templates/page/index.html.eex [hunk 2]
```diff
@@ -37,3 +54,9 @@
     </ul>
   </div>
 </div>
+
+<script type='text/javascript'>
+  $('.logout-link').click(function() {
+    $.ajax({url:"/logout", method: "DELETE"}).done(function() { window.location.reload(); });
+  });
```
Comment (tylerwillingham) [inline example_app/web/templates/page/index.html.eex:61]: I'm curious - is there a benefit to using this approach over something like `<%= link "Logout", to: logout_path(@conn, :delete), method: :delete %>` ? The existing approach (and yours) certainly has more straight forward markup.

Comment (steveklebanoff) [inline example_app/web/templates/page/index.html.eex:61]: `<%= link "Logout", to: logout_path(@conn, :delete), method: :delete %>`  looks cleaner, but my understanding is that the `logout_path` action just returns JSON and will not trigger a redirect after the logout is successful.   We need to specify a callback to reload the page once the logout is done, see `.done(function() { window.location.reload(); });` above.  I don't believe there is way to do this using the `link` function.  I'm open to feedback and other ideas though

Comment (tylerwillingham) [inline example_app/web/templates/page/index.html.eex:61]: That makes sense, I didn't realize that was the case. I should look at the example app a little closer.

File: example_app/web/templates/page/index.html.eex [hunk 3]
```diff
@@ -7,24 +19,29 @@
...
-      <li>
-        <a href="#" onclick='javascript:$.ajax({url:"/logout", method: "DELETE"})'>Logout</a>
-      </li>
-      <li>
-        <a href="/recover_password">Recover Password</a>
-      </li>
-      <li>
-        <a href="/reset_password">Reset Password</a>
-      </li>
+      <%= if Addict.Helper.current_user(@conn) do %>
```
Comment (trenpixster) [inline example_app/web/templates/page/index.html.eex:22]: Can you please change to `Addict.Helper.is_logged_in/1` here instead?

**Comment by trenpixster** [comment]
Awesome, thanks! It'd be great to also update the Readme to point out the use of the helper functions :)

**Comment by steveklebanoff** [comment]
Updated to use new helper.  Not sure exactly what to put in README.  Hopefully this is good to go -- happy to make any other changes if needed though.

**Comment by trenpixster** [comment]
Thanks for this! :thumbsup:

**Pull request merged**